### ISSUE TITLE
error ValidatePdfSignature

### ISSUE THREAD

**Issue by edwinmacalopu** [lsnepomuceno/laravel-a1-pdf-sign#27]
an error occurs when validating a signature, apparently the format for other countries is different, it is a certificate from Peru

```
try {
           dd(ValidatePdfSignature::from('.../sign.pdf'));
        } catch (\Throwable $th) {
           return $th;
        }
```
adobe acrobat 
![image](https://user-images.githubusercontent.com/15043775/147263691-c9935129-5eb7-4640-80cf-26142b2c091e.png)

use ValidatePdfSignature()

![image](https://user-images.githubusercontent.com/15043775/147263304-6da69a5a-b235-42c4-810a-4001ac06e51d.png)

**Comment by lsnepomuceno** [comment]
Hi @edwinmacalopu,

Would you have a signed example pdf file?

That way I can validate the structure.

**Comment by edwinmacalopu** [comment]
@lsnepomuceno  yes I attached signed PDF  .
[sign.pdf](https://github.com/lsnepomuceno/laravel-a1-pdf-sign/files/7770527/sign.pdf)

**Comment by edwinmacalopu** [comment]
@lsnepomuceno find the error is a field that when separating with explode does not have key 1 value in the signature, in my case I put a validator if there is key 1 add it to the array
![image](https://user-images.githubusercontent.com/15043775/147274290-0698eed2-4230-4a8b-b731-3ecbcfcdd736.png)

```
private function processDataToInfo(string $data): array
  {
    $data = explode('/', trim($data));
    $finalData = [];
    foreach ($data as $info) {
      $infoTemp = explode('=', trim($info));
        if (isset($infoTemp[1])) {
        $finalData[] = [$infoTemp[0] => $infoTemp[1]];
       }
      }
    return $finalData;
  }
```

![image](https://user-images.githubusercontent.com/15043775/147276542-4f3a7ef4-29ce-4188-ba9c-d1fac331df9f.png)

**Comment by lsnepomuceno** [comment]
@edwinmacalopu 

Although I haven't identified the problem with validating array indexes (as shown in the screenshots below), I've updated the method to avoid potential issues.

Available in version [0.0.14](https://github.com/lsnepomuceno/laravel-a1-pdf-sign/releases/tag/0.0.14).

Thank you for your contribution.

![validate-2](https://user-images.githubusercontent.com/14093492/147290299-bbdd147f-5b36-4b11-b4ca-cd8e3b997c6d.png)
![validate-1](https://user-images.githubusercontent.com/14093492/147290300-65e0a8fc-14c9-47c6-8889-22e530f0d68e.png)
